torch/csrc/distributed/autograd/functions/SendRpcBackwards.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <torch/csrc/autograd/function.h>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+struct TORCH_API SendRpcBackwards : public torch::autograd::Node {
```

Comment:
Should this be called `SendRpcBackward` instead? Not 100% sure, just want to make sure we are consistent with other backward node.